llama/patches/0001-ggml-backend-malloc-and-free-using-the-same-compiler.patch

```diff
@@ -111,23 +110,11 @@ index d9110491..37ee2a6d 100644
  }
  
  static void * ggml_cuda_host_malloc(size_t size) {
-diff --git a/ggml/src/ggml-metal/ggml-metal.m b/ggml/src/ggml-metal/ggml-metal.m
-index cb8eff4a..7bccc7bf 100644
---- a/ggml/src/ggml-metal/ggml-metal.m
-+++ b/ggml/src/ggml-metal/ggml-metal.m
-@@ -6032,6 +6032,7 @@ static void ggml_backend_metal_buffer_free_buffer(ggml_backend_buffer_t buffer)
-     }
- 
-     free(ctx);
-+    free(buffer);
```

Comment:
I think this got moved to ggml-metal.cpp and we still need to free the buffer in both of the private and shared _free_buffer() functions. I believe that we should actually be using delete rather than free here, though there is no practical difference in this case.

Reply:
fixed